data/modules/Assassination/Assassination.lua

```diff
@@ -9,6 +9,7 @@ local Comms = require 'Comms'
 local Timer = require 'Timer'
 local Event = require 'Event'
 local Mission = require 'Mission'
+local MissionUtils = require "modules.MissionUtils"
```

Comment:
As a matter of convention, we prefer to use single-quotes for module names, especially when omitting the parentheses to `require()`.